irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel where all players and contributors can discuss the game or request support.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#Mminetest-Dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        "Development channel, strictly engine or minetest_game development talk only; note that webchat is muted due to too much spam/flood."
```

Comment:
quotes not needed

Reply:
Typo, they shouldn't be there, will fix.